### PR TITLE
Unify demo exec

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -124,12 +137,38 @@ at ``/opt/lsst/software/stack``:
...
+
+.. code-block:: sh
+
+    vagrant destroy -f el7
+
+
+Preserve instance disk state and shutdown
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Vagrant instances, or "boxes" in vagrant-speaker, which have been ``halt``-ed can be restarted by running ``vagrant up ....``.
```
Comment (jonathansick) [inline index.rst]: Vagrant-speak

**Comment by jonathansick** [comment]
On line 53 'vagrant' can be capitalized to Vagrant
https://github.com/lsst-sqre/sqr-002/pull/2/files#diff-db4b047b30ffef7855b12b527860ac2cR53

**Comment by jonathansick** [comment]
I like the re-org to run the demo with either workflow. Looks good to me.

**Comment by jhoblitt** [comment]
@jonathansick thanks for the review!

**Pull request merged**